src/app/api/marp/route.ts

```
@@ -53,8 +54,11 @@ export async function POST(request: NextRequest) {
           );
         }
 
+        // Handle language-specific slide files
+        const slideFilePath = slideFile.endsWith('.md') ? `src/slides/${slideFile}` : `src/slides/${slideFile}.md`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)